_data/reports/2021-09-defiprotocol.md

```diff
@@ -989,7 +989,7 @@ _Submitted by WatchPug_
 
 <https://github.com/code-423n4/2021-09-defiProtocol/blob/main/contracts/contracts/Auction.sol#L143>
 
-```solidity=140
```

Comment:
The semantics of some of these are a little unclear -- the blocks sometimes refer to lines outside of the (optional?) mention above them.